### ISSUE TITLE
Scoreboard for players on the creator mode room

### ISSUE THREAD

**Issue by selimdoyranli** [RadKod/parolla#47]
Need a scoreboard for players on the creator mode room

- The player writes a nickname before starting the game.
- After finishing the game, the player's nickname, room information, and player score are sent to the backend API.
- The backend API records this information and creates a "players' scores" list.
- Other players in the room will see the scores in a user interface component using the list obtained from the backend.

**Comment by erayaydin** [comment]
Nice to have:
- Recognize the nickname for other playing sessions.
- Check nickname availability for leaderboard (w/ backend services)
- Show leaderboard only for completed games (motivate player to finish daily)

Also, this feature can easily be implemented with leaderboard services.

**Comment by apo-bozdag** [comment]
we can do it

**Comment by erayaydin** [comment]
> we can do it

![image](https://user-images.githubusercontent.com/5854916/230900576-2ec405ec-2115-42a3-9cf0-4e910a86278e.png)
